recipes/clustergrammer/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: clustergrammer
+  version: "1.13.5"
+
+source:
+  fn: clustergrammer-1.13.5.tar.gz
+  url: https://pypi.python.org/packages/f4/ad/c0d0fd6e1c917d51df418d084201ef87c9b17288284870a31ac3ecebd571/clustergrammer-1.13.5.tar.gz
+  md5: d7b600b34c5674710f83efbeddbc50f7
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scikit-learn
+    - pandas
+
+  run:
+    - libgcc
```

Comment:
Is libgcc really needed here?

Reply:
it was when I tested locally.

Reply:
as in, that's the only way it passed the mulled tests (and I tried a bunch of things)

Reply:
Yeah, this can be. If so then please add `- libgcc  # [linux]`

Reply:
ok! thank you